packages/cli/test/acceptance/app-run.acceptance.js

```diff
@@ -35,37 +35,68 @@ skipIf(process.env.CI == null, describe, 'app-generator (SLOW)', () => {
     outdir: path.join(sandboxDir, 'sandbox-app'),
   };
 
-  before('scaffold a new application', createAppProject);
-  /** @this {Mocha.Context} */
-  async function createAppProject() {
-    // Increase the timeout to 1 minute to accommodate slow CI build machines
-    this.timeout(60 * 1000);
-    await helpers
-      .run(appGenerator)
-      .inDir(appProps.outdir)
-      // Mark it private to prevent accidental npm publication
-      .withOptions({private: true})
-      .withPrompts(appProps);
-  }
-
-  before('install dependencies', installDependencies);
-  /** @this {Mocha.Context} */
-  async function installDependencies() {
-    // Run `lerna bootstrap --scope @loopback/sandbox-app --include-filtered-dependencies`
-    // WARNING: It takes a while to run `lerna bootstrap`
-    this.timeout(15 * 60 * 1000);
-    process.chdir(rootDir);
-    await lernaBootstrap(null, appProps.name);
-  }
-
-  it('passes `npm test` for the generated project', /** @this {Mocha.Context} */ function () {
+  before(
+    'scaffold a new application',
+    /** @this {Mocha.Context} */ function () {
+      // Increase the timeout to 1 minute to accommodate slow CI build machines
+      this.timeout(60 * 1000);
+
+      return (
+        helpers
+          .run(appGenerator)
+          .inDir(appProps.outdir)
+          // Mark it private to prevent accidental npm publication
+          .withOptions({private: true})
+          .withPrompts(appProps)
+      );
+    },
+  );
+
+  before(
+    'install dependencies',
+    /** @this {Mocha.Context} */ function () {
+      // Run `lerna bootstrap --scope @loopback/sandbox-app --include-filtered-dependencies`
+      // WARNING: It takes a while to run `lerna bootstrap`
+      this.timeout(15 * 60 * 1000);
+      process.chdir(rootDir);
+      return lernaBootstrap(null, appProps.name);
+    },
+  );
+
+  it('passes `npm test` for the generated project', /** @this {Mocha.Context} */ async function () {
     // Increase the timeout to 5 minutes,
     // the tests can take more than 2 seconds to run.
     this.timeout(5 * 60 * 1000);
 
-    return new Promise((resolve, reject) => {
+    await new Promise(resolve => {
```

Comment:
Please add a comment explaining why we run 3 steps below instead of `npm test`?

- npm run pretest
- npm run test --ignore-scripts
- npm run posttest

Reply:
Done